src/system/api/nameGeneration.ts

```diff
@@ -155,8 +155,11 @@ export class nameGeneration {
 
     let names = nameJournal?.data.content;
     // TODO: This could possibly be more robust to allow users to not have to be exact in entering names
-    names = names?.replace("<pre>", "").replace("</pre>", "");
-    const nameArray = names?.split("<br />");
+    names = names
```

Comment:
@wake42  something appears to have changed with how JournalEntry items have pre-formatted text formatted.  It used to use "<br />" for line breaks but now seems to be "\n".  I left in a replace of "<br />" to "\n" since some older JournalEntries still had "<br />" and it can't hurt.  It then splits on "\n".

Reply:
Let's just roll with this and if we have to fix it later, so be it.